### ISSUE TITLE
Search unit tests fail on Windows

### ISSUE THREAD

**Issue by joaomoreno** [Microsoft/vscode#31635]
Both win 32 and 64 builds are blocked by this test:

```
2017-07-28T08:38:38.3419886Z   1) QueryBuilder parseSearchPaths absolute includes:
2017-07-28T08:38:38.3419886Z 
2017-07-28T08:38:38.3419886Z       AssertionError: 2 == 1
2017-07-28T08:38:38.3419886Z       + expected - actual
2017-07-28T08:38:38.3419886Z 
2017-07-28T08:38:38.3419886Z       -2
2017-07-28T08:38:38.3419886Z       +1
2017-07-28T08:38:38.3419886Z       
2017-07-28T08:38:38.3419886Z       at assertEqualSearchPathResults (file:///D:/work/agent1/1/s/out-build/vs/workbench/parts/search/test/common/queryBuilder.test.js:296:16)
2017-07-28T08:38:38.3419886Z       at testIncludes (file:///D:/work/agent1/1/s/out-build/vs/workbench/parts/search/test/common/queryBuilder.test.js:63:17)
2017-07-28T08:38:38.3419886Z       at testIncludesDataItem (file:///D:/work/agent1/1/s/out-build/vs/workbench/parts/search/test/common/queryBuilder.test.js:67:17)
2017-07-28T08:38:38.3419886Z       at Array.forEach (native)
2017-07-28T08:38:38.3419886Z       at Context.<anonymous> (file:///D:/work/agent1/1/s/out-build/vs/workbench/parts/search/test/common/queryBuilder.test.js:134:19)
```

In order to get builds for testing today, I will comment the test out temporarily.

**Comment by joaomoreno** [comment]
Commented the test out: 0a1b82ca3c0de6376673279362f340b6937a72a8

**Comment by roblourens** [comment]
Fixed in e900e7d12847962a99466e727245eeccd5c9387d, apologies
